apps/web-giddh/src/app/home/components/profit-loss/profit-loss.component.scss

```diff
@@ -0,0 +1,59 @@
+@import "../../../../assets/css/color.scss";
+.total-amount {
+    font-size: 18px;
+}
+
+.total-income-text {
+    color: $cornflower-blue;
+}
+
+.total-expenses-text {
+    color:$goldenrod
+}
+
+span.month-dropdown {
+    font-size: 12px;
+    color: $dark-gray;
+    vertical-align: top;
+}
+
+.dashboard-filter .btn-group {
+    display: block;
+    line-height: 1;
+}
+
+.dashboard-filter .btn-group span {
+    vertical-align: middle;
+    color: $dark-gray;
+}
+
+.underline {
```

Comment:
underline class is already created pls remove it

Reply:
I am not write this new css, just separating css from TS file